corehq/apps/api/odata/utils.py

```diff
@@ -43,11 +43,15 @@ def get_properties_by_xmlns(domain, app_id, xmlns):
     form_export_schema = get_latest_form_export_schema(
         domain, app_id, complete_xmlns
     ) or FormExportDataSchema.generate_schema_from_builds(domain, app_id, complete_xmlns)
-    export_items = [
-        item for item in form_export_schema.group_schemas[0].items
-        if isinstance(item, ExportItem)
-    ]
-    return set([get_odata_property_from_export_item(item) for item in export_items]) - {''}
+
+    if not form_export_schema.group_schemas:
```

Comment:
Is this a good spot for a `try/except` instead?

Reply:
I see how it could be more readable but because the error is a general ```IndexError``` I'm concerned that a different error with the same exception type might fail silently.